packages/shared/tailwind.config.js

```diff
@@ -154,6 +154,8 @@ module.exports = {
     },
     screens: {
       mobileL: '420px',
+      mobileXL: '500px',
+      mobileXXL: '550px',
```

Comment:
Ugh, not your fault but this is pain to my eyes 🤣 
@nensidosari Is the idea to eventually remove/change the breakpoints once we find experiment winner?

Reply:
The idea was to have these for the experiment, depending on how that goes yes they will be removed or kept. Maybe we should leave a todo note here to remember to remove these if the experiment is not a winner. 

@DragosIorgulescu can u add the TODO comment?